lib/dep-graph-builders/pnpm/types.ts

```diff
@@ -66,3 +67,13 @@ export type PnpmImporters = Record<
   PnpmWorkspacePath,
   Record<DepName, DepVersion>
 >;
+
+export type ScannedPnpmProject = {
+  packageManager: string;
```

Comment:
we can enforce some specific values for this type (enum maybe?)

Reply:
i updated with a union there